Gruntfile.js

```diff
@@ -176,6 +179,23 @@ module.exports = function (grunt) {
         grunt.file.write(manifestPath, JSON.stringify(manifest));
     });
 
+    grunt.registerTask('core', function () {
+        if (version) {
+            let done = this.async();
+            resolveRuntimeVersion(version).then(v => {
+                if(!grunt.file.exists(`${process.env.localAppData}/OpenFin/runtime/${v}`)){
+                    console.log(v);
+                    grunt.fail.fatal('The specified version of the core was not found');
```

Comment:
It would be good to either keep going but just warn that we're using the default core, or preferably to download the runtime. That can be saved for a follow-up pr though.